tests/converter/test_code_generation.py

```diff
@@ -86,45 +86,65 @@ def dict_codegen_segmentation(cpu_name) -> dict:
     configurations = []
-    configurations.extend(get_configurations_by_test_cases(test_cases, dict_codegen_classification(cpu_name)))
-    configurations.extend(get_configurations_by_test_cases(test_cases, dict_codegen_classification_resnet(cpu_name)))
-    configurations.extend(get_configurations_by_test_cases(test_cases, dict_codegen_object_detection(cpu_name)))
-    configurations.extend(get_configurations_by_test_cases(test_cases, dict_codegen_segmentation(cpu_name)))
+    configurations.extend(get_configurations_by_test_cases(test_cases,
+                                                           dict_codegen_classification(cpu_name, use_fpga)))
+    configurations.extend(get_configurations_by_test_cases(test_cases,
+                                                           dict_codegen_classification_resnet(cpu_name, use_fpga)))
+    configurations.extend(get_configurations_by_test_cases(test_cases,
+                                                           dict_codegen_object_detection(cpu_name, use_fpga)))
+    configurations.extend(get_configurations_by_test_cases(test_cases, dict_codegen_segmentation(cpu_name, use_fpga)))
 
     return configurations
 
 
 def get_configurations_x86_64():
-    cpu_name = "x86_64"
+    cpu_name = "x86"
+    use_fpga = "disable"
     test_cases = [
-        {'use_avx': True, 'hard_quantize': True, 'threshold_skipping': True},
-        {'use_avx': True, 'hard_quantize': True, 'threshold_skipping': False},
-        {'use_avx': True, 'hard_quantize': False, 'threshold_skipping': False},
-        {'use_avx': False, 'hard_quantize': True, 'threshold_skipping': True},
-        {'use_avx': False, 'hard_quantize': True, 'threshold_skipping': False},
-        {'use_avx': False, 'hard_quantize': False, 'threshold_skipping': False},
+        {'hard_quantize': True, 'threshold_skipping': True},
+        {'hard_quantize': True, 'threshold_skipping': False},
+        {'hard_quantize': False, 'threshold_skipping': False},
     ]
-    configurations = get_configurations_by_architecture(test_cases, cpu_name)
+    configurations = get_configurations_by_architecture(test_cases, cpu_name, use_fpga)
 
-    additional_test_configuration = updated_dict(dict_codegen_classification(cpu_name),
+    additional_test_configuration = updated_dict(dict_codegen_classification(cpu_name, use_fpga),
                                                  {'use_run_test_script': True})
     additional_test_cases = [
-        {'use_avx': True, 'input_name': 'raw_image.png', 'test_level': TEST_LEVEL_FUTURE_TARGET},
-        {'use_avx': True, 'input_name': 'preprocessed_image.npy', 'from_npy': True},
-        {'use_avx': False, 'input_name': 'raw_image.png', 'test_level': TEST_LEVEL_FUTURE_TARGET},
-        {'use_avx': False, 'input_name': 'preprocessed_image.npy', 'from_npy': True},
+        {'input_name': 'raw_image.png', 'test_level': TEST_LEVEL_FUTURE_TARGET},
+        {'input_name': 'preprocessed_image.npy', 'from_npy': True},
+    ]
```

Comment:
Why are `test_cases` and `additional_test_cases` separated?

Reply:
I'm sorry but I'm not sure why they are separated because it's code of someone else.
I checked the code again and maybe the difference is which way is used for the test.
First one calls dynamic library from test code directory.
Second one use run.py (not directly) to call dynamic library for the test.

Reply:
Thank you! I've understood.